searx/settings.yml

```diff
@@ -67,11 +67,13 @@ ui:
 #    key : !!binary "your_morty_proxy_key"
 
 outgoing: # communication with search engines
-    request_timeout : 2.0 # default timeout in seconds, can be override by engine
+    request_timeout : 3.0 # default timeout in seconds, can be override by engine
```

Comment:
Why is the timeout increased? Isn't this going to lead to searx returning results more slowly from users' point of view?

Reply:
I have forgotten to reset the value from my configuration.

Done in the last commit.